components/director/internal/destinationfetchersvc/client.go

```diff
@@ -43,6 +43,7 @@ type DestinationServiceAPIConfig struct {
 	PagingCountParam              string        `envconfig:"APP_DESTINATIONS_PAGE_COUNT_PARAM,default=$pageCount"`
 	PagingCountHeader             string        `envconfig:"APP_DESTINATIONS_PAGE_COUNT_HEADER,default=Page-Count"`
 	SkipSSLVerify                 bool          `envconfig:"APP_DESTINATIONS_SKIP_SSL_VERIFY,default=false"`
+	OAuthTokenPath                string        `envconfig:"APP_DESTINATION_OAUTH_TOKEN_PATH,default=/oauth/token"`
```

Comment:
I saw that in the charts we also use `/oauth/token` but as far as I remember in the external-services-mock we do not have such endpoint? We have `/secured/oauth/token` for the unsecured server and `/cert/token` for the secured one. 

The interesting part is all of the tests are passing so I might be missing something. Do you know the exact URL that is called in the local setup? or which endpoint/handler of the external services mock is invoked in this case?

Reply:
ah, I overlook the local overrides, sorry. I guess we're using this approach to override only the local setup because the productive one is with `/oauth/token` and that way avoiding "productive" override?